NEWS

```diff
@@ -7,13 +7,177 @@
 ##
 #
 
-## Current
+## 1.12.0 RC1 (December 2, 2021)
 ### Features:
+#### Core
+* Added initial support for Go language bindings
+* Added memory invalidation on error detection
+* Added threshold for ep connection matching in UCP
+* Added new objects to VFS (md, component, log_level, etc)
+* Added config variable to specify what loadable modules are needed
+* Added sigaction-override config to disable sigaction overriding
 #### UCP
-* Added API for querying UCP library attributes
+* Added new sockaddr private data format
+* Enabled rendezvous and tag sync for all cases with error handling
+* Added iov zcopy support to RMA operations
+* Added usage of mpool set for unexpected eager message to reduce memory consumption
+* Added client_id to ucp_worker_create() and ucp_conn_request_query() APIs
+* Added support for modifying UCT and UCS configs by ucp_config_modify() API
+* Added address versioning to correctly preserve wire compatibility
+* Optimized unpacked rkeys memory consumption
+* Added request flag to influence latency vs. bandwidth protocol
+* Added ucp_worker_address_query() API
+* Reduced memory management overhead with new protocols
+* Updated ucp_ep_query() API with getting local and remote addresses
+* Improved performance calculations in new protocols
+* Added AMO support with new protocols
+* Added stages for rendezvous in new protocols
+* Added put|get_zcopy and pipeline based rendezvous in new protocols
+* Added support for user-defined alignment in AM
+* Added support for offload tag sync in new protocols
+* Updated ucp_atomic_post() to use NBX flow
+#### UCT
+* Added stub for memory invalidation support in CMA
+* Added IPv6 address support in TCP
+* Added latency estimation to uct_iface_estimate_perf()
+* Adjusted knem and cma overhead cost
+* Added API - uct_iface_is_reachable_v2()
+* Increased built-in TCP keep-alive interval to 2 seconds
+#### RDMA CORE (IB, ROCE, etc.)
+* Added check for CQ overrun in assert mode
+* Added bitmap usage for releasing detached DCIs
+* Added config for requests ack frequency with DevX
+* Added remote QP info to tx error CQE traces
 #### UCS
-* Added API to Read boot ID value or use machine_guid
+* Added memory pool set data structure
+* Added more accurate topology distance estimation
+* Added thread-safe put to ptr_map
+* Added prints of leaked callbacks from the callback queue
+* Added new ptr_array API for bulk allocation
+* Added ucs_string_buffer_append_flags() for string buffer
+* Added ucs_ffs32()
+* Removed a diagnostic message when fuse thread is stopped
+* Added ucs_vsnprintf_safe() which always adds '\0'
+* Added API for a per-process aggregate-sum statistics report
+* Added configurable limit for the memory consumed by rcache
+* Added configuration for VFS(FUSE) thread affinity
+* Added memory limit support to memtrack
+#### CUDA
+* Added option to set cuda_copy bandwidth
+* Added profiling of CUDA runtime function calls
+* Added stub for memory invalidation support
```

Comment:
I'm not sure if we need to list "stub" functions - seems like does not contribute to anything (you have few assurances here).

Reply:
this is important change, because it allows these transports to be selected for RNDV protocol.
Remove?

Reply:
I would rephrase this:  Added stub for memory invalidation support ignorer to enable CUDA transport selection for rendezvous protocols

Reply:
wdyt `Added stub for memory invalidation support ignorer to enable CUDA transport selection for rendezvous protocols in case of error handling enabled`?

Reply:
👍

Reply:
Actually,  I would make it a bit shorter `Added stub for memory invalidation support to enable CUDA-IPC transport selection for rendezvous protocols`

Reply:
i would remove this altogether, it's part of memory invalidation err flows fix